app/models/concerns/project_search.rb

```diff
@@ -51,6 +52,14 @@ def exact_name
       name
     end
 
+    def maven_name
```

Comment:
I wonder if there's a more generic thing to think about here given things like the `@foo/bar` style npm packages, etc.

Reply:
Those ones work, I've only noticed a problem with Java so far. https://libraries.io/search?q=cli-plugin-eslint That being said, not an expert with elasticsearch so I don't know if there's a better way.

Reply:
This is the right approach... I just think that we want to do it a little more generally. And actually thinking about it, we probably want both groupId and artifactId to get indexed nicely.

I wonder if the thing to do is to have a field that's `extra_names` (or something -- I don't like that name but my ☕️ hasn't kicked in yet) which is an array of additional searchable words for a package. And then on a per ecosystem basis we return what those would be